src/main/java/com/redhat/exhort/providers/JavaMavenProvider.java

```diff
@@ -115,7 +115,7 @@ private Sbom buildSbomFromTextFormat(Path textFormatFile) throws IOException {
     String[] array = new String[lines.size()];
     lines.toArray(array);
     //    createSbomIteratively(lines,sbom);
-    parseDependencyTree(root, 0, array, sbom);
+    parseDependencyTree(root, 0, array, sbom, null);
```

Comment:
If you set the scope to `null` it will always be the same value. You're not distinguishing between required an optional or I haven't seen it in the current changes.

Reply:
This SBOM scope enhancement is only added to Gradle, not yet in maven provider in this implementation.